src/node.h

```diff
@@ -169,10 +169,6 @@ NODE_EXTERN v8::Local<v8::Value> MakeCallback(
 
 }  // namespace node
 
-#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
-#include "node_internals.h"
```

Comment:
I don't think this will work on Windows as you are not including `uv.h` (which then includes `uv-win.h`) in `node_main.cc` anymore.